### PR TITLE
Create mail_and_packages.json

### PR DESCRIPTION
**Pull request by firstof9**
Add mail_and_packages.json

### CODE REVIEW

File: components/mail_and_packages.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "mail_and_packages",
+    "owner": ["@moralmunky", "@firstof9"],
+    "manifest": "https://raw.githubusercontent.com/moralmunky/Home-Assistant-Mail-And-Packages/0.3.0/custom_components/mail_and_packages/manifest.json",
```
Comment (ludeeus) [inline components/mail_and_packages.json:4]: This should point to your development branch, not a specific tag.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: components/mail_and_packages.json (hunk 1 above)
Comment (firstof9) [reply components/mail_and_packages.json:4]: 0.3.0 is the current dev branch.

Comment (ludeeus) [reply components/mail_and_packages.json:4]: Yeah, was default branch i meant

Comment (firstof9) [reply components/mail_and_packages.json:4]: So the manifest needs to be under the default branch not the dev branch or is this ok?

Comment (ludeeus) [reply components/mail_and_packages.json:4]: The one that will keep being updated, i highly doubt that is 0.0.3

Comment (firstof9) [reply components/mail_and_packages.json:4]: Alright thanks ludeeus, I'll fix it up once we cut a new latest.

**Comment by firstof9** [comment]
Looks fixed now, thanks again for your patience @ludeeus

**Review by frenck — APPROVED** [review]
Thanks, @firstof9 👍

**Pull request merged**